Source/MoistureMeter/MeadowApp.cs

```diff
@@ -28,7 +29,7 @@ public override Task Initialize()
 
             DisplayController.Instance.Initialize(projectLab.Display);
 
-            sensor = new MoistureSensor(projectLab.GroveAnalog.Pins.D0);
+            sensor = new MoistureSensor(projectLab.GroveAnalog.Pins.D0.CreateAnalogInputPort(1));

```

Comment:
Is this necessary? App ran fine without creating here, the driver does it internally for me.